tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/util/tls/CertificatesUtils.java

```diff
@@ -0,0 +1,138 @@
+package org.jboss.intersmash.tools.util.tls;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.openshift.OpenShifts;
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.api.model.SecretBuilder;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class CertificatesUtils {
```

Comment:
I'd prefer to keep them as separate classes: this make it easier to understand what's going on with certificates which is a delicate issue especially when debugging some error or making some modification

Reply:
+1 about the principle, but are qe creating duplicated code this way?

Reply:
So, if there's duplicated code (e.g.: processCall() or something like that) it will be refactored into an helper. This can be done asynchronously, provided it's properly tracked by a new issue.